examples/media-services/non_terraform_managed_storage/main.tf

```diff
@@ -0,0 +1,25 @@
+# Can you also get the id by adding a azurerm_storage_account that references the
+# same properties as the existing storage account.
+variable "existing_storage_account_id" {
+  default = "/subscriptions/00000000-0000-0000-0000-000000000000/resourcegroups/testrg/providers/Microsoft.Storage/storageAccounts/teststorage"
```

Comment:
rather than hard-coding this ID (which we'd recommend against doing) - could we instead use [the Storage Account Data Source](https://www.terraform.io/docs/providers/azurerm/d/storage_account.html) instead?